sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -545,6 +582,9 @@ def to_open_api(self) -> OpenApiModel:
                 )
                 for experiment_mapping in self.experiment_mappings
             ],
+            route_name_path=self.route_name_path
+            if self.route_name_path is not None
+            else "",
```

Comment:
I believe an empty string "" is required because the API validator requires this field to be set right? Also, in what cases will this value be `None`, asking this because the default value should already be "" based on how you initialise this class, so I think this is not required?

Reply:
I have the same question about why we need to manipulate the value here - can we just use `None` for experiment_mappings and route_name_path if they are not set? The API doesn't require them to be sent in, right?

Reply:
> I believe an empty string "" is required because the API validator requires this field to be set right?

> can we just use None for experiment_mappings and route_name_path if they are not set? The API doesn't require them to be sent in, right?

Yeah the OpenAPI classes needs some value to be set and we can't just pass a `None` value directly. One way to go around this is to selectively pass non-`None` values into the `kwargs` dict:

```python
    def to_open_api(self) -> OpenApiModel:
        kwargs = {}
        if self.experiment_mappings is not None:
            kwargs["experiment_mappings"] = self.experiment_mappings
        if self.route_name_path is not None:
            kwargs["route_name_path"] = self.route_name_path

        return turing.generated.models.EnsemblerStandardConfig(
            **kwargs,
        )
```

I've refactored the `StandardRouterEnsemblerConfig` class a little so we aren't manipulating the values of `route_name_path` in the `to_open_api` method but I've also changed the default values of `experiment_mappings` and `route_name_path` members to be `None`. 

> Also, in what cases will this (`route_name_path`) value be None[?]

In general, one of `route_name_path` or `experiment_mappings` will be `None` but not the other. This corresponds to the scenario when the user only wants the standard ensembler to be explicitly by the experiment mappings OR the route name path. 

Note also that the SDK doesn't perform any validations to ensure one and only of these two fields are set, but the API will perform this and return an error if this property is violated.

Reply:
> Yeah the OpenAPI classes needs some value to be set and we can't just pass a None value directly.

I've managed to change this by setting the `nullable` field as `True` within the OpenAPI specs for both `experiment_mappings` and `route_name_path`.